models/user.go

```diff
@@ -224,16 +236,36 @@ func (u *User) GetEmail() string {
 
 // APIFormat converts a User to api.User
 func (u *User) APIFormat() *api.User {
+	apiURL := setting.AppURL + "api/v1/users/" + u.LowerName
+	publicRepoCount, _ := GetPublicRepositoryCount(u)
 	return &api.User{
-		ID:        u.ID,
-		UserName:  u.Name,
-		FullName:  u.FullName,
-		Email:     u.GetEmail(),
-		AvatarURL: u.AvatarLink(),
-		Language:  u.Language,
-		IsAdmin:   u.IsAdmin,
-		LastLogin: u.LastLoginUnix.AsTime(),
-		Created:   u.CreatedUnix.AsTime(),
+		ID:               u.ID,
+		UserName:         u.Name,
+		FullName:         markup.Sanitize(u.FullName),
```

Comment:
Why is this necessary? Doesn't the JSON layer take care of this?

Reply:
I didn't right that. This was just moved from convert.go so that both models/user.go and api/v1/convert/covert.go use the same implementation.

Reply:
Hmm, I think you're right though. It was copied from convert.go since I first did my changes there and then moved that whole structure population into here. Will remove.